data/src/main/java/com/netflix/iceberg/data/IcebergGenerics.java

```diff
@@ -59,13 +60,25 @@ public ScanBuilder where(Expression rowFilter) {
       return this;
     }
 
+    public ScanBuilder caseInsensitive() {
```

Comment:
Other places use `caseSensitive(boolean)`. I typically like to add both variants and I think it would be a good idea if it isn't difficult.

Reply:
Re `IcebergGenerics`, I was confused on when a caller should use that instead of `Table.newScan()` since `TableScan` allows you to refine?

Reply:
`IcebergGenerics` is a builder, not a refinement pattern. It is used to build and execute a scan. We use this for our Java client, where users read directly from tables. The builder pattern is better for users. (The refinement pattern is good for passing partially configured scans to other components.)

Reply:
> Other places use caseSensitive(boolean). I typically like to add both variants and I think it would be a good idea if it isn't difficult.

Just to make sure I follow, you suggest that we add `caseSensitive(boolean)` to IcebergGenerics while keeping `caseInsensitive()`, correct?

If so, do you suggest we also add `caseInsensitive()` to all other interfaces that have `caseSensitive(boolean)`?

Reply:
Yes, both. I think it is good to have a couple of versions of methods like these that set booleans. For example, `Tasks` exposes [`throwFailureWhenFinished()`](https://github.com/apache/incubator-iceberg/blob/master/core/src/main/java/com/netflix/iceberg/util/Tasks.java#L108-L121) as well as `suppressFailureWhenFinished()` and `throwFailureWhenFinished(boolean)` that all control a boolean variable. That allows passing a boolean through using the last case, but also makes it easy to set the option to a constant.

This is a minor update and not that important, though.

Reply:
Got it. Linking these thread to Issue #83 so that we don't loose track. Will fix in subsequent PR.